crates/aranya-daemon/src/api.rs

```diff
@@ -302,6 +363,7 @@ impl DaemonApiHandler {
                 Effect::QueryAqcNetIdentifierResult(_) => {}
                 Effect::QueriedLabelAssignment(_) => {}
                 Effect::QueryLabelExistsResult(_) => {}
+                Effect::QueryAqcNetworkNamesOutput(_) => {}
```

Comment:
This is closer to the naming convention we already have:
```suggestion
                Effect::QueryAqcNetworkNameResult(_) => {}
```
Only a single net identifier can be returned per effect.

Reply:
Saving it for that separate PR we discussed, I mostly care about the functionality in this